servant-server/test/Servant/Server/UsingConfigSpec/CustomCombinator.hs

```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE PolyKinds #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+
+-- | This is a custom combinator for S.S.UsingConfigSpec. It's split up into
+-- its own module to be able to test how exactly module import work when using
+-- the config.
+module Servant.Server.UsingConfigSpec.CustomCombinator where
+
+import           Servant
+import           Servant.Server.Internal.Config
+import           Servant.Server.Internal.RoutingApplication
+
+data CustomCombinator (tag :: k)
+
+instance forall subApi (c :: [*]) tag .
```

Comment:
True. Fixed here: eb20151. I wanted to put that kind signature somewhere for clarity, but I put it further down.